File: brian_ortega_delahoz/Clases/Programación/POO/Julio2024/29-Jul-24/class.dart
class Persona {
  String? mombre;
  int? edad;
  double? estatura;
}

class Videojuego {
  String? mombre;
  String? genero;
  String? tipo;
}

class Celular {
  String? marca;
  String? modelo;
  String? color;
}

void main() {
  Persona brian = Persona();
  brian.mombre = "Brian";
  brian.edad = 20;
  brian.estatura = 1.69;

  Videojuego fortnite = Videojuego();
  fortnite.mombre = "Fortnite";
  fortnite.genero = "Battle Royal";
  fortnite.tipo = "Accion";

  Celular xiaomi = Celular();
  xiaomi.marca = "Xiaomi";
  xiaomi.modelo = "Redmi";
  xiaomi.color = "Blanco";
}

File: brian_ortega_delahoz/Clases/Programación/POO/Julio2024/29-Jul-24/class.rb
class Persona
    attr_accessor :nombre, :edad, :estatura
  end
  
  class Videojuego
    attr_accessor :nombre, :genero, :tipo
  end
  
  class Celular
    attr_accessor :marca, :modelo, :color
  end
  
  persona = Persona.new
  persona.nombre = 'Brian'
  persona.edad = 20
  persona.estatura = 1.69
  
  videojuego = Videojuego.new
  videojuego.nombre = 'Fortnite'
  videojuego.genero = 'Battle Royal'
  videojuego.tipo = 'Accion'
  
  celular = Celular.new
  celular.marca = 'Xiaomi'
  celular.modelo = 'Redmi'
  celular.color = 'Blanco'
  